docs/lib/auth/fragments/js/advanced.md

```diff
@@ -242,30 +242,31 @@ const SignInWithGoogle = () => {
 
 ### Using Amplify UI components
 
-To enable social sign-in in your app with Identity Pools, add `Google client_id`, `Facebook app_id` and/or `Amazon client_id` properties to `Authenticator` component. This will create a sign in button when rendering the `Authenticator` in your app.
+To enable social sign-in in your app with Identity Pools, add `Google client_id`, `Facebook app_id` and/or `Amazon client_id` properties to the `AmplifyAuthenticator` component. This will create a sign in button when rendering the `AmplifyAuthenticator` in your app.
 
 ```javascript
-import { Authenticator } from 'aws-amplify-react/dist/Auth';
+import { AmplifyAuthenticator } from '@aws-amplify/ui-react';
 
 const federated = {
-    google_client_id: '',
-    facebook_app_id: '',
-    amazon_client_id: ''
+    googleClientId: '',
+    facebookAppId: '',
+    amazonClientId: ''
 };
 
 return (
-    <Authenticator federated={federated}>
+    <AmplifyAuthenticator federated={federated}>
 )
 ```
 
 Or you can use it with `withAuthenticator`:
 ```js
+import { withAuthenticator } from '@aws-amplify/ui-react';
 const AppWithAuth = withAuthenticator(App);
 
 const federated = {
-    google_client_id: '', // Enter your google_client_id here
-    facebook_app_id: '', // Enter your facebook_app_id here
-    amazon_client_id: '' // Enter your amazon_client_id here
+    googleClientId: '', // Enter your googleClientId here
+    facebookAppId: '', // Enter your facebookAppId here
+    amazonClientId: '' // Enter your amazonClientId here
```

Comment:
Just a minor nit -- can we have theses comments on the code block above? Otherwise LGTM!

Reply:
Done!